crates/cast/src/helpers/scarb_utils.rs

```diff
@@ -153,41 +146,23 @@ pub fn get_package_metadata<'a>(
     Ok(package)
 }
 
-pub fn parse_scarb_config(
+pub fn get_scarb_config(
```

Comment:
Let's put this to `impl CastConfig`, rename it to `from_package_metadata` and make it return `Option`. Then we could do `unwrap_or_default` on it. 
WDYT?

Reply:
We would lose our error propagating code in that case as well